crawl-ref/source/dat/des/arrival/simple.des

```diff
@@ -2029,3 +2029,80 @@ x.G.G.G.x
 x.......x
 xxxx+xxxx
 ENDMAP
+
+###############################################################################
+
+NAME:   giann_horns   
+TAGS:   arrival no_monster_gen
+ORIENT: float
+MAP
+x@xx...................xx@x
+x..xx.................xx..x
+x...xx..G..xx+xx..G..xx...x
+x....xx...xx...xx...xx....x
+x.....xxxxx.....xxxxx.....x
+x............{............x
+xx.......................xx
+ xxxxxxxxxxxx+xxxxxxxxxxxx
+ENDMAP
+
+NAME:   giann_bridges
```

Comment:
Looks fine to me. The terrain is uncomfortably open, giving the player nowhere to hide, but at least if a large group of monsters try to engage you have pseudo-corridors to work with.